config/vufind/permissions.ini

```diff
@@ -48,6 +48,8 @@
 ; user        - Grant the permissions to logged in users whose user attribute match
 ;               the given regular expression pattern. For valid pattern syntax see
 ;               http://php.net/manual/de/reference.pcre.pattern.syntax.php.
+; cookie      - Grant the permissions to logged in users whose cookie value match the
```

Comment:
The rest of this list is alphabetical, so it may be best to move this new entry to the top.

Reply:
I'm also not sure if the description is entirely accurate. Would this be closer to the current reality of what is happening?

```
Grant the permissions to users who have the named cookie(s) set in their browser.
```

As noted elsewhere, depending on if/how we incorporate value checks into the process, we will want to expand this description to either include information about how to specify values or else warnings about the security limitations of an approach based purely on cookie name.

Reply:
thanks @demiankatz I have re-ordered and reworded the description in the permissions.ini file in my latest follow-up.

more follow-ups to come.